### ISSUE TITLE
[2.9.0] UnicodeEncodeError raised on create_filtered_path

### ISSUE THREAD

**Issue by nMustaki** [scoutapp/scout_apm_python#417]
Hi,

so we still use Python2, and got an UnicodeErrorException in create_filtered_path, ligne 53: 

```python
return path + "?" + urlencode(filtered_params)
```

filtered_params value:
```python
filtered_params = [[u'search', u'Clos du Chêne, Montévrain, France'],]
```
query_params has the same value

As it is the output from Sentry, I do not know whether the strings were really Unicode as implied by the "u".

**Comment by adamchainz** [comment]
Hi @nMustaki 

Thanks for the error report. Can you provide some of the traceback? I can probably fix the error from the information you've provided but it would be helpful to see a bit more information.

I'll have a look this evening.

Thanks,

Adam

**Comment by nMustaki** [comment]
I'm copying it through Sentry, I hope it's enough ?

UnicodeEncodeError: 'ascii' codec can't encode character u'\xea' in position 10: ordinal not in range(128)

```python
# urllib.py in urlencode at line 1342
    v = quote_plus(str(v))
# scout_apm/core/web_requests.py in create_filtered_path at line 53
    return path + "?" + urlencode(filtered_params)
# scout_apm/django/middleware.py in track_request_view_data at line 119
    path, [(k, v) for k, vs in request.GET.lists() for v in vs]
# scout_apm/django/middleware.py in process_view at line 226
    track_request_view_data(request, tracked_request)
# django/core/handlers/base.py in _get_response at line 178
    response = middleware_method(request, callback, callback_args, callback_kwargs)
# django/core/handlers/exception.py in inner at line 41
    response = get_response(request)
```

**Comment by nMustaki** [comment]
Many thanks for the speedy update!

**Comment by adamchainz** [comment]
No worries. Released on PyPI as version 2.9.1:  https://pypi.org/project/scout-apm/2.9.1/ . Enjoy!
